### PR TITLE
Add tax configuration migration guide

### PR DESCRIPTION
**Pull request by krzysztofzuraw**

### CODE REVIEW

File: docs/upgrade-guides/3-18-to-3-19.mdx [hunk 1]
```diff
@@ -98,3 +98,15 @@ sequenceDiagram
...
     App -->- Customer: Order details
 ```
+
+## Taxes configuration
+
+In previous versions (legacy tax flow), users could select to use the Tax app or flat rates. This means that if there are multiple tax applications or plugins enabled in your environment, Saleor will attempt to use them, which can result in taxes not being properly configured. Because of that, we added an option to select a specific method of tax calculation. To change it:
+
+1. Login to your dashboard
+2. Go to `Configuration -> Taxes` setting page
+3. If you have legacy flow - dashboard will display warning message under select component for tax calculation method. Warning message will be displayed for a given channel or for country exception as well.
```
Comment (peelar) [inline docs/upgrade-guides/3-18-to-3-19.mdx:108]: **style (non-blocking)**:

```suggestion
3. If you have legacy flow, dashboard will display warning message under select component for tax calculation method. Warning message will be displayed for a given channel or for country exception as well.
```

File: docs/upgrade-guides/3-18-to-3-19.mdx [hunk 2]
```diff
@@ -98,3 +98,15 @@ sequenceDiagram
...
+4. To migrate to new tax flow select one of options from dropdown below warning:
```
Comment (peelar) [inline docs/upgrade-guides/3-18-to-3-19.mdx:109]: **style (non-blocking)**:

```suggestion
4. To migrate to new tax flow, select one of options from the dropdown below the warning:
```

**Review by peelar — APPROVED** [review]

File: docs/upgrade-guides/3-18-to-3-19.mdx [hunk 3]
```diff
@@ -98,3 +98,15 @@ sequenceDiagram
     Saleor -->>- App: Order details
     App -->- Customer: Order details
 ```
+
+## Taxes configuration
```
Comment (timuric) [inline docs/upgrade-guides/3-18-to-3-19.mdx:102]:
```suggestion
## Taxes Configuration
```

**Review by timuric — COMMENTED** [review]
Please adjust all titles to title case 🙏

**Review by Droniu — DISMISSED** [review]

File: docs/upgrade-guides/3-18-to-3-19.mdx [hunk 4]
```diff
@@ -98,3 +98,15 @@ sequenceDiagram
     Saleor -->>- App: Order details
     App -->- Customer: Order details
 ```
+
+## Taxes Configuration
+
+In previous versions (legacy tax flow), users could select to use the Tax app or flat rates. This means that if there are multiple tax applications or plugins enabled in your environment, Saleor will attempt to use them, which can result in taxes not being properly configured. Because of that, we added an option to select a specific method of tax calculation. To change it:
```
[the remark this answers is not in the document]
Comment (krzysztofzuraw) [reply docs/upgrade-guides/3-18-to-3-19.mdx]: Fixed 🙇🏻

**Review by Cloud11PL — APPROVED** [review]

**Review by poulch — APPROVED** [review]

**Pull request merged**